<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="launch">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seed">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>200</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="seed_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="from_file">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="default_file">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Default file</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="user_given">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>User-given</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="size_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Game size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="size">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>45</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
